the check for having atomics checks only load, but apparently store needs
-latomic while load doesn't

diff --git a/source/Utility/CMakeLists.txt b/source/Utility/CMakeLists.txt
index cc0bf5f..26bf8b0 100644
--- a/source/Utility/CMakeLists.txt
+++ b/source/Utility/CMakeLists.txt
@@ -19,9 +19,10 @@ if (CMAKE_SYSTEM_NAME MATCHES "Windows")
   list(APPEND LLDB_SYSTEM_LIBS ws2_32 rpcrt4)
 endif ()
 
-if (NOT HAVE_CXX_ATOMICS64_WITHOUT_LIB )
-    list(APPEND LLDB_SYSTEM_LIBS atomic)
-endif()
+# if (NOT HAVE_CXX_ATOMICS64_WITHOUT_LIB )
+#     list(APPEND LLDB_SYSTEM_LIBS atomic)
+# endif()
+list(APPEND LLDB_SYSTEM_LIBS atomic)
 
 add_lldb_library(lldbUtility
   ArchSpec.cpp
